.github/workflows/node.js.yml

```
@@ -0,0 +1,29 @@
+name: ash-nazg tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Start CouchDB
+      uses: iamssen/couchdb-github-action@master
```

```suggestion
      uses: iamssen/couchdb-github-action@0.3.0
```
should we use a specific version (in this case 0.3.0) rather than @master?

---

Indeed, that might be safer